crates/re_renderer/shader/depth_cloud.wgsl

```diff
@@ -66,13 +58,30 @@ struct VertexOut {
     @location(3) point_radius: f32,
 };
 
+// ---
+
+struct PointData {
+    pos_in_world: Vec3,
+    unresolved_radius: f32,
+    color: Vec4
+}
+
 // Backprojects the depth texture using the intrinsics passed in the uniform buffer.
 fn compute_point_data(quad_idx: i32) -> PointData {
     let wh = textureDimensions(depth_texture);
     let texcoords = IVec2(quad_idx % wh.x, quad_idx / wh.x);
 
     // TODO(cmc): expose knobs to linearize/normalize/flip/cam-to-plane depth.
-    let world_space_depth = depth_cloud_info.world_depth_from_texture_value * textureLoad(depth_texture, texcoords, 0).x;
+    var world_space_depth = depth_cloud_info.world_depth_from_texture_value * textureLoad(depth_texture, texcoords, 0).x;
+
+    if (world_space_depth <= 0.0 || f32max <= world_space_depth) {
```

Comment:
This expression should probably be inverted to catch any NaNs (even though we can assume they don't exist… O.o)